### PR TITLE
feat: add settings payload domain definition

### PR DESCRIPTION
**Pull request by gorjan5sk**

### CODE REVIEW

File: packages/settings/src/Domain/Setting/SettingName.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/**
+ * Enumerate all Settings with their names as identifiers
```
Comment (karolsojko) [inline packages/settings/src/Domain/Setting/SettingName.ts:2]: Not necessary comment. Does not add value.

File: packages/settings/src/Domain/Setting/SettingPayload.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { SettingName } from './SettingName'
+
+/**
+ * Describe Setting payloads (contents) matched with Setting enum keys
+ *
```
Comment (karolsojko) [inline packages/settings/src/Domain/Setting/SettingPayload.ts:5]: Same

**Review by karolsojko — APPROVED** [review]
Could you just remove the comments? They don't add any value in here

**Comment by gorjan5sk** [comment]
@karolsojko removed comments. Merging! :)

**Pull request merged**